includes/abstracts/abstract-wc-stripe-connect-rest-controller.php

```diff
@@ -94,7 +94,7 @@ public function get_internal( $request ) {
 
 		$this->prevent_route_caching();
 
-		return $this->get( $request );
+		return $this->get( $request ); // @phpstan-ignore-line
```

Comment:
Anywhere we are ignoring something, would be nice to add a short comment explaining why

Reply:
In addition, now looking at the rest of the changes here, most of what we're doing is just ignoring lines. I'm not super familiar with PHPStan but assuming it's similar to PHPCS, are there consistent things we are ignoring such that we could ignore those globally, in some sort of config file? This would prevent us from littering the code with a bunch of ignore statements